examples/tutorials/Advanced/Websockets - Federated training/README.md

```diff
@@ -0,0 +1,25 @@
+# Federated learning using websockets - MNIST example
+
+The scripts in this folder let you execute a federated training via three websocket connections.
+
+The script start_websocket_servers.py will start the Websocket server workers for Alice, Bob and Charlie.
+```
+$ python start_websocket_servers.py
+```
+
+The training is then started by running the script run_websocket_client.py:
+```
+$ python run_websocket_client.py
+```
+This script
+ * loads the MNIST dataset,
+ * distributes it onto the three workers
```

Comment:
Could we do this step by intitializing each of alice, bob and charlie with the data parameter. That seems to be much more in line with how fl will work in the wild

Reply:
This could be a separate example, where the focus is on setting up the remote data in a realistic way. My suggestion is to do a separate pull request for this.